src/core/server/services/events/comments.ts

```diff
@@ -57,6 +57,21 @@ export function publishCommentCreated(
   }
 }
 
+export function publishCommentReleased(
+  publish: Publisher,
+  comment: Pick<Comment, "id" | "storyID" | "parentID" | "status">
+) {
+  if (!comment.parentID && hasPublishedStatus(comment)) {
```

Comment:
This seems to filter by only top level comments, is this intentional?

Reply:
yeah because the event for commentCreated filters by top-level so I figured we'd want to keep that experience the same